### DISCUSSION TITLE
Black stroke

### DISCUSSION THREAD

**Discussion by mykhailo2674823** [Alex313031/Thorium-Win#316] [Q&A] [answered]
How do i remove this black color stroke on tabs in the latest version at this point?
![Screenshot1](https://github.com/user-attachments/assets/a5f5f9cc-6570-4dc6-8593-56924e39cbd4)

**Comment by gz83** [comment, marked as answer]
Disable tab outline in chrome://flags

**Reply by mykhailo2674823** [reply]
Thanks, it's working
